src/components/Upload/table.tsx

```diff
@@ -43,8 +57,99 @@ export default function UploadTable({
         }))
       );
     }
-  }, [meta.keys]);
+    if (getMeta.hasOwnProperty("headings")) {
+      setCsvHeader(
+        getMeta.headings.map((o, id) => ({
+          id,
+          text: o,
+          key: o,
+          fieldName: o,
+          name: o,
+          minWidth: 160,
+          isResizable: true,
+          isPadded: false
+        }))
+      );
+    }
+    setData(csvData);
+  }, [meta.keys][getMeta.headings]);
+  const _getErrorMessageLat = value => {
+    let regex = /^[0-9]*\.[0-9]/;
+    if (!value.match(regex)) {
+      setAllFilesUploaded(false);
+      return "Numbers is allowed with decimal point";
+    } else {
+      setAllFilesUploaded(true);
+    }
+  };
 
+  const _getErrorMessageLong = value => {
+    let regex = /^[0-9]*\.[0-9]/;
+    if (!value.match(regex)) {
+      setAllFilesUploaded(false);
+      return "Numbers is allowed with decimal point";
+    } else {
+      setAllFilesUploaded(true);
+    }
+  };
+
+  const displayTable = (item, row, column) => {
+    const fieldContent = item[column.fieldName];
+    if (column.key === "latitude" && row !== 0 && row !== 1) {
+      return (
+        <TextField
+          name={column.id}
+          onChange={(e, v) => {
+            setLatColumn(row, v);
+          }}
+          onGetErrorMessage={_getErrorMessageLat}
+          defaultValue={row == 1 ? "latitude" : fieldContent}
+          validateOnLoad={false}
+        />
+      );
+    } else if (column.key === "longitude" && row !== 0 && row !== 1) {
+      return (
+        <TextField
+          name={column.id}
+          onChange={(e, v) => {
+            setLongColumn(row, v);
+          }}
+          onGetErrorMessage={_getErrorMessageLong}
+          defaultValue={row == 1 ? "longitude" : fieldContent}
+          validateOnLoad={false}
+        />
+      );
+    }
```

Comment:
- [ ] Duplicate code, Simplify Logic
- [ ] and avoid hard condtiions/magic numbers like `0` and `1` in conditions